### ISSUE TITLE
Valid handle with empty baseURL

### ISSUE THREAD

**Issue by Bara** [ErikMinekus/sm-ripext#40]
That's not really important, but I think that's worth to mention. With empty base url the returned handle is valid, but if we use this handle with simple Get, it returns status code 0 with empty error message. That's bit confusing while debugging.

Example plugin
```sp
#include <sourcemod>
#include <ripext>

public void OnPluginStart()
{
    HTTPClient httpClient = new HTTPClient("");
    
    PrintToServer("Hande valid: %d", (httpClient != null));

    httpClient.Get("Test", GetCallback);
}

public void GetCallback(HTTPResponse response, any value, const char[] error)
{
    PrintToServer("Status: %d, Error: %s", response.Status, error);
}
```

Output is
```
sm plugins load test
Hande valid: 1
Status: 0, Error:
```

**Comment by ErikMinekus** [comment]
It looks like libcurl will do this with any invalid URL. As URL validation is complex, I don't think it's worth the time to deal with this.

**Comment by Bara** [comment]
How about a simple string length check, as example strings with a length of 2 or lower are invalid.

**Comment by ErikMinekus** [comment]
I think libcurl will happily accept `"a"`, if there is such a hostname. But I will throw an exception when the base URL is empty, that makes sense.

**Comment by ErikMinekus** [comment]
This was fixed in 6e7dc5a, I plan to release it soon.
